static/sass/_v1_pattern_phone_contact.scss

```diff
@@ -0,0 +1,22 @@
+// Resource box style patterns
+@mixin ubuntu-p-phone-contact {
+  @include p-phone-contact;
+}
+
+@mixin p-phone-contact {
+  .p-phone-contact {
```

Comment:
With the icon-and-text pattern, I think this could be more generic. I'll refactor it.